package.json

```diff
@@ -60,7 +60,7 @@
     "dotenv": "^2.0.0",
     "ejs": "^2.4.2",
     "express": "^4.14.0",
-    "hull": "0.11.6",
+    "hull": "^0.11.7",
```

Comment:
@mkawalek let's stick with fixed hull-node version to have full control over which version we are using.